src/common/components/vanilla/list/list-icon.js

```diff
@@ -0,0 +1,17 @@
+import React, { PropTypes } from 'react';
+import styles from './list-icon.css';
+
+export default function ListWithIcon(props) {
+  const className = props.className || '';
+
+  return (
+    <ul className={`${styles.listIcon} ${className}`}>
```

Comment:
In case you're not aware, there's a handy utility in the repo for stuff like this, though we usually use it once it starts to get really complex, but worth knowing about anyway: https://github.com/JedWatson/classnames